ui-lib/src/main/java/co/electriccoin/zcash/ui/MainActivity.kt

```diff
@@ -150,6 +159,54 @@ class MainActivity : FragmentActivity() {
         }
     }
 
+    private fun setContent(
```

Comment:
We'll extract this new logic into a separate PR and provide documentation for it.

Reply:
done